src/main/java/picard/fingerprint/FingerprintMetrics.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2019 The Broad Institute
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package picard.fingerprint;
+
+import htsjdk.samtools.metrics.MetricBase;
+import picard.analysis.FingerprintingDetailMetrics;
+import picard.analysis.FingerprintingSummaryMetrics;
+
+/**
+ * Class for holding metrics on a single fingerprint.
+ * Note: this is distinct from {@link FingerprintingDetailMetrics} and
+ * {@link FingerprintingSummaryMetrics} in that it is calculated on a single fingerprint,
+ * and attempts to describe how likely that fingerprint is to have arisen from an actual sample
+ * as opposed to having artifacts such as contamination, or strong bias towards homozygous genotypes.
```

Comment:
Can you note here that "expected" is based on the allele frequency from the `HAPLOTYPE_MAP`? I suppose this is noted in the doc for `CalculateFingerprintMetrics`, but it would be nice to also note it here since the docs for the metrics file aren't next to the docs for the tool.